lib/c-api/src/wasm_c_api/mod.rs

```diff
@@ -120,6 +156,12 @@ pub unsafe extern "C" fn wasm_instance_new(
 #[no_mangle]
 pub unsafe extern "C" fn wasm_instance_delete(_instance: Option<Box<wasm_instance_t>>) {}
 
+// TODO: NOT part of the standard Wasm C API
+#[no_mangle]
+pub unsafe extern "C" fn wasm_instance_get_vmctx_ptr(instance: &wasm_instance_t) -> *mut c_void {
```

Comment:
Where / How this is used?

Reply:
It's needed to call the trampolines directly -- currently I'm using it in a test C file I made that I didn't commit

Reply:
Why someone would call the trampolines directly? I think it's better to enforce users to use the normal wasm-c-api to call wasm functions.
Thoughts @MarkMcCaskey